_config.yml

```diff
@@ -11,8 +11,8 @@ discord_link:       https://discord.gg/2tWhanq
 github_link:        https://github.com/UWB-ACM
 linkedin_link:      https://www.linkedin.com/in/uwbacm/
 
-# Add your google-analytics ID here to activate google analytics
-google_analytics:   UA-XXXXXXXXX-X # out your google-analytics code
+# Google-analytics ID
```

Comment:
can you add a comment here that says that this value is also hard-coded in `google-analytics.html` ?

Reply:
fixed!